### ISSUE TITLE
pythondistdeps: handle ~= version range

### ISSUE THREAD

**Issue by ignatenkobrain** [rpm-software-management/rpm#616]
The Bodhi has requirement written like `pyramid~=1.7`. Unfortunately, pythondistdeps doesn't handle this.

It makes RPM to produce following error:

```
error: Dependency tokens must begin with alpha-numeric, '_' or '/': ~= 1.7
```

**Comment by Conan-Kudo** [comment]
Hmm, so according to [PEP 440](https://www.python.org/dev/peps/pep-0440/#compatible-release), this is equivalent to `(pyramid >= 1.7 with pyramid < 2.0)`. So it's equivalent to `^` in Rust and Ruby.

**Comment by ffesti** [comment]
This is fixed by ed864b8842
